packages/xo-web/src/common/intl/messages.js

```diff
@@ -937,6 +937,8 @@ const messages = {
   noNbdConnection: 'No NBD Connection',
   nbdConnection: 'NBD Connection',
   insecureNbdConnection: 'Insecure NBD Connection (not allowed through XO)',
+  // ----- Pool patches tab -----
+  isSingleHost: "Rolling pool update can only work when there's multiple hosts in a pool with a shared storage",
```

Comment:
Try to get a key that fit more the message content.

Reply:
What about poolUpdateSharedStorage ? Or multiHostPoolUpdate
I think it's too long but I don't have other ideas ....